k8s/base-with-cert/app/route-staff.yaml

```diff
@@ -1,4 +1,6 @@
-- op: add
-  path: /metadata/annotations
-  value:
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: aspace-staff
+  annotations:
```

Comment:
Changing the patch strategy here to use a strategic merge, so that the allow list annotation is retained after this patch, i.e. the `annotations` object is merged. https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/#patch-using-path-strategic-merge